pkg/apis/nmstate/v1alpha1/nodenetworkconfigurationpolicy_types.go

```diff
@@ -46,7 +46,7 @@ type NodeNetworkConfigurationPolicySpec struct {
 // NodeNetworkConfigurationPolicyStatus defines the observed state of NodeNetworkConfigurationPolicy
 // +k8s:openapi-gen=true
 type NodeNetworkConfigurationPolicyStatus struct {
-	Enactments EnactmentList `json:"enactments,omitempty" optional:"true"`
+	Enactments NodeNetworkConfigurationPolicyList `json:"enactments,omitempty" optional:"true"`
```

Comment:
This is not the final state, is it?

Reply:
This is wrong that's why we have enactments in the CRD, removed it already.